stream/src/main/java/org/apache/kafka/stream/processor/PConfig.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.stream.processor;
+
+public class PConfig {
+
+    private String name;
+    private Object value;
+
+    public PConfig(String name, Object value) {
+        this.name = name;
+        this.value = value;
+    }
+
+    public Object value() {
```

Comment:
If a user want to pass some parameter to KafkaProcessor, he/she has to use PConfig and cast value() to whatever the user expects. This forces a user to write a type-unsafe code.